src/Control/Monad/Aff/Class.purs

```diff
@@ -13,35 +12,34 @@ import Control.Monad.RWS.Trans (RWST)
 import Control.Monad.State.Trans (StateT)
 import Control.Monad.Trans.Class (lift)
 import Control.Monad.Writer.Trans (WriterT)
-
 import Data.Monoid (class Monoid)
 
-class MonadEff eff m <= MonadAff eff m | m -> eff where
-  liftAff :: forall a. Aff eff a -> m a
+class MonadEff eff m ⇐ MonadAff eff m | m → eff where
+  liftAff ∷ ∀ a. Aff eff a → m a
 
-instance monadAffAff :: MonadAff e (Aff e) where
+instance monadAffAff ∷ MonadAff e (Aff e) where
   liftAff = id
 
-instance monadAffContT :: MonadAff eff m => MonadAff eff (ContT r m) where
+instance monadAffContT ∷ MonadAff eff m ⇒ MonadAff eff (ContT r m) where
```

Comment:
Same as my previous comment about pseudo-ligatures, although here I think it's particularly tricky due to how small the character is for `⇒` compared to the rest of the text, it makes it much harder to see.